app/routes.py

```diff
@@ -1,2 +1,78 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response, request
+from .models.planet import Planet
+from app import db
 
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+#---HELPER FUNCTIONS---
+def validate_planet(planet_id):
+    try:
+        planet_id = int(planet_id)
+    except:
+        abort(make_response({"message": f"Planet {planet_id} invalid"}, 400))
+
+    planet = Planet.query.get(planet_id)
+
+    if not planet:
+        abort(make_response({"message": f"Planet {planet_id} not found"}, 404))
+
+    return planet
+
+
+#---ROUTES---
+@planets_bp.route("", methods=["POST"])
+def create_planet():
+    request_body = request.get_json()
+    new_planet = Planet.new_instance_from_dict(request_body)
```

Comment:
Some input validation on the request body might be useful here (Checking to see if name and description are present for example).